formidling/src/main/java/no/nav/ung/sak/formidling/vedtak/regler/strategy/EndringInntektFullUtbetalingStrategy.java

```diff
@@ -17,7 +18,7 @@ public EndringInntektFullUtbetalingStrategy() {
 
     @Override
     public VedtaksbrevStrategyResultat evaluer(Behandling behandling, LocalDateTimeline<DetaljertResultat> detaljertResultat) {
-        return new VedtaksbrevStrategyResultat(null, "Ingen brev ved full utbetaling etter kontroll av inntekt.", IngenBrevÅrsakType.IKKE_RELEVANT);
+        return new VedtaksbrevStrategyResultat(null, null, "Ingen brev ved full utbetaling etter kontroll av inntekt.", IngenBrevÅrsakType.IKKE_RELEVANT);
```

Comment:
Vurder å bruke statisk metode i staden for bedre lesbarhet (VedtaksbrevStrategyResultat.ingenBrev())